### PR TITLE
IOS-655: App sometimes crashes if I set off all 'stream sources' and click between tabs

### PR DESCRIPTION
**Pull request by MiloszFilimowski**
### Ticket
[IOS-655](https://netguru.atlassian.net/browse/IOS-655)

### Task Description
App sometimes crashes if I set off all 'stream sources' and click between tabs (it crashes on central tab with inbbbox icon) 
http://take.ms/ITXxq

### CODE REVIEW

File: Inbbbox/Source Files/Managers/Shots State Handlers/ShotsInitialAnimationsStateHandler.swift [hunk 1]
```diff
@@ -45,7 +45,7 @@ class ShotsInitialAnimationsStateHandler: NSObject, ShotsStateHandler {
     }
     
     var shouldShowNoShotsView: Bool {
-        return shotsCollectionViewController?.shots.count == 0 && Settings.areAllStreamSourcesOff()
+        return Settings.areAllStreamSourcesOff()
```
Comment (pikor) [inline Inbbbox/Source Files/Managers/Shots State Handlers/ShotsInitialAnimationsStateHandler.swift]: I might be wrong but I think that during initial animations this kind of messages should not be visible.

Comment (MiloszFilimowski) [inline Inbbbox/Source Files/Managers/Shots State Handlers/ShotsInitialAnimationsStateHandler.swift]: What do you mean? If all stream sources are off I think this is the proper message to display.

Comment (pikor) [inline Inbbbox/Source Files/Managers/Shots State Handlers/ShotsInitialAnimationsStateHandler.swift]: Even during initial state? If yes, than it's okay to merge.

Comment (MiloszFilimowski) [inline Inbbbox/Source Files/Managers/Shots State Handlers/ShotsInitialAnimationsStateHandler.swift]: Yes even than.

**Pull request merged**